### ISSUE TITLE
ui: SQL Activity on active statements has results with no query

### ISSUE THREAD

**Issue by j82w** [cockroachdb/cockroach#96177]
This was hit while running the insights workload. The query is empty on the page. If you go to details it says the statement can not be found. Several of the details are loaded like start time, but the query itself is not found.

./cockroach workload init insights "connectionString" --db-count=10  --table-count=50

./cockroach workload run insights "connectionString" --db-count=10  --table-count=50

<img width="1983" alt="Screen Shot 2023-01-30 at 9 49 25 AM" src="https://user-images.githubusercontent.com/8868107/215510352-bf6784dc-d939-460b-b068-4218e41aea0f.png">

<img width="2268" alt="Screen Shot 2023-01-30 at 9 49 39 AM" src="https://user-images.githubusercontent.com/8868107/215510317-70616fea-6ddf-4637-9520-787138c6f6cd.png">

**Comment by xinhaoz** [comment]
I don't think this is a bug, I think there was just no statement executed yet for these txns (I see is the txns page).

**Comment by kevin-v-ngo** [comment]
We should have a better message to indicate that the first statement has not ran yet. Or remove the transaction entirely from this view.
